src/main/java/io/fusionauth/domain/provider/NintendoApplicationConfiguration.java

```diff
@@ -34,6 +34,9 @@
   @InternalJSONColumn
   public String client_secret;
 
+  @InternalJSONColumn
+  public IdentityProviderOauth2Configuration oauth2 = new IdentityProviderOauth2Configuration().with(c -> c.uniqueIdClaim = "id");
```

Comment:
I don't think we need to pull this object in, this has a lot of other fields that will duplicate what is in this class. 

All we need is `emailClaim`, `uniqueIdClaim`, and `usernameClaim`.